jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -0,0 +1,132 @@
+lib = library(identifier: "jenkins@20211118", retriever: legacySCM(scm))
```

Comment:
We'll need to add this job to https://github.com/opensearch-project/opensearch-build/issues/1468 once PR is merged, so we can add test cases for this when `libTesters` are ready